shared/gh/js/views/gh.series-info.js

```diff
@@ -159,58 +155,56 @@ define(['gh.core', 'gh.constants', 'moment', 'clickover'], function(gh, constant
     };
 
 
-    ///////////////
-    //  POPOVER  //
-    ///////////////
+    /////////////
+    //  MODAL  //
+    /////////////
 
     /**
-     * Set up and show the series popover
+     * Set up and show the series info modal
      *
      * @private
      */
-    var setUpSeriesPopover = function() {
+    var setUpModal = function() {
         var $trigger = $(this);
 
         // Store the series ID
         var seriesId = $trigger.data('id');
-
-        // Render the popover window
-        gh.utils.renderTemplate($('#gh-series-info-popover-template'), {
-            'data': {
-                'id': seriesId
+        // Retrieve the series title
+        var seriesTitle = $trigger.closest('.list-group-item').find('.gh-list-description-text').html();
+
+        // Retrieve the tripos data
+        var _triposData = gh.utils.triposData();
+
+        /* Retrieve the tree structure, populated with the organisational unit*/
+        var parent = null;
+        var partId = History.getState().data.part;
+        _.each(_triposData, function(orgUnitType) {
+            var _part = _.find(orgUnitType, function(orgUnit) {
+                return orgUnit.id === partId;
+            });
+            if (_part) {
+                parent = _part;
             }
-        }, $('.gh-series-info-popover-container[data-id="' + seriesId + '"]'));
-
-        // Show the popover window
-        var $content = $trigger.closest('.gh-list-group-item-container').find('.popover.info[data-id="' + seriesId + '"]');
+        });
 
-        var options = {
-            'class_name': 'gh-series-popover gh-series-info-popover',
-            'container': 'body',
-            'content': $content.html(),
-            'global_close': true,
-            'html': true,
+        // Generate the hierarchy string
+        var hierarchy = gh.utils.renderHierarchyString(parent, ' > ');
```

Comment:
Although actually that's not consistent with the designs for agenda view. I'll leave it up to you to decide which looks better.